client/asset/btc/btc.go

```diff
@@ -5474,6 +5711,75 @@ func (btc *baseWallet) BondsFeeBuffer() uint64 {
 	return 15 * bondsFeeBuffer(btc.segwit, btc.feeRateLimit()) / 10
 }
 
+// FundMultiOrder funds multiple orders in one shot. MaxLock is the maximum
+// amount that the wallet can lock for these orders. If maxLock == 0, then
+// there is no limit. An error is returned if the wallet does not have enough
+// available balance to fund each of the orders, however, if splitting is
+// not enabled and all of the orders cannot be funded due to mismatches in
+// UTXO sizes, the orders that can be funded are funded. The values in the
+// MultiOrder parameter are funded in the order they are passed in.
+func (btc *baseWallet) FundMultiOrder(mo *asset.MultiOrder, maxLock uint64) ([]asset.Coins, [][]dex.Bytes, uint64, error) {
```

Comment:
In `The values in the MultiOrder parameter are funded in the order they are passed in.`, does that guarantee that it stops on the first failed order?  I ask because I'm pretty sure the caller of `FundMultiOrder` will want to know the values of the orders that it was able to fund, and if it skips to the next value if one fails, you won't know what you got funded.  Can the doc be clarified?

Reply:
Yes it does guarantee that. I'll update docs.